query-service-impl/src/main/java/org/hypertrace/core/query/service/projection/ProjectionTransformation.java

```diff
@@ -417,4 +432,14 @@ private Filter updateFilterForComplexAttributeExpressionFromFilter(Filter origin
         .map(QueryRequestUtil::createContainsKeyFilter)
         .collect(Collectors.toList());
   }
+
+  private String getOriginalKey(AttributeExpression attributeExpression) {
+    String alias = attributeExpression.getAlias();
```

Comment:
We have a function for this now I believe, can share it right?

Reply:
That accepts expression.